### PR TITLE
Remove tabs from examples (issue #225)

### LINKED ISSUE #225: Remove tab characters from Example 7
See https://github.com/w3c/imsc/issues/224#issuecomment-290434056

### PR DESCRIPTION
**Pull request by palemieux**
Closes #225

### CODE REVIEW

File: imsc1/spec/examples/fillLineGap-example.xml [hunk 1]
```diff
@@ -7,25 +7,25 @@
...
+        <layout>
+            <region xml:id="bottom" tts:origin="10% 10%" tts:extent="80% 80%" tts:displayAlign="after" />
+        </layout>
+    </head>
+    <body>
+        <div>
+            <p xml:id="subtitle1" region="bottom" begin="00:00:00.000" end="00:00:30.000" style="paragraphStyle">
+                <span style="spanStyle">##Line gaps##</span><br/>
+                <span style="spanStyle">The quick </span><span style="spanStyleBig">brown</span><span style="spanStyle"> fox</span><br/>
+                <span style="spanStyle">jumps over the </span><span style="spanStyleSmall">lazy</span><span style="spanStyle"> dog</span><br/>
```
Comment (nigelmegitt) [inline imsc1/spec/examples/fillLineGap-example.xml]: This appears still to have white space at the end of the line - I thought the idea here is to remove that? Same on lines 11 and 14 though that has less impact.

Comment (palemieux) [inline imsc1/spec/examples/fillLineGap-example.xml]: The objective was to remove "tab" characters. I have not trimmed spaces, which should have no impact AFAIK.

Comment (nigelmegitt) [inline imsc1/spec/examples/fillLineGap-example.xml]: As discussed today, the purpose of this document is not to test behaviour with white space at the ends of lines so it should be removed.

Comment (palemieux) [inline imsc1/spec/examples/fillLineGap-example.xml]: See revised PR

**Pull request merged**